src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -267,7 +267,7 @@ class DefaultColorControlSceneHandler : public scenes::DefaultSceneHandlerImpl
         ReturnErrorOnFailure(pair_iterator.GetStatus());
 
         // Switch to the mode saved in the scene
-        if (SupportsColorMode(endpoint, targetColorMode))
+        if (SupportsColorMode(endpoint, static_cast<EnhancedColorModeEnum>(targetColorMode)))
```

Comment:
Shouldn't `targetColorMode` just be `EnhancedColorModeEnum`?  Again, followup is ok.

Reply:
Updated ... but now given these changes I actually wonder if this code is buggy as it uses identical checks and callbacks for both enhanced and non-enhanced modes.

Added a TODO ....

Reply:
It seems that the color control header had its own copy that used different naming (even the legacy naming for extra confusion) on types. Did a major update to only use the enhanced names.